frontends/election-manager/src/app.test.tsx

```diff
@@ -1253,3 +1253,95 @@ test('super admin Smartcards screen navigation', async () => {
   userEvent.click(screen.getByText('Create Election Cards'));
   screen.getByRole('heading', { name: 'Election Cards' });
 });
+
+test('super admin smartcard modal', async () => {
+  jest.useFakeTimers();
+  (areVvsg2AuthFlowsEnabled as jest.Mock).mockImplementation(() => true);
+
+  const card = new MemoryCard();
+  const hardware = MemoryHardware.buildStandard();
+  const storage = await createMemoryStorageWith({
+    electionDefinition: eitherNeitherElectionDefinition,
+  });
+  render(<App card={card} hardware={hardware} storage={storage} />);
+  await authenticateWithSuperAdminCard(card);
+
+  let modal: HTMLElement;
+  const blankCard = undefined;
+  const programmedCard1 = {
+    t: 'admin',
+    h: eitherNeitherElectionDefinition.electionHash,
+    p: '123456',
+  } as const;
+  const programmedCard2 = {
+    t: 'superadmin',
+  } as const;
```

Comment:
I think there are test utils for making cards in `test-utils/smartcards`

Reply:
Ah perfect!